backend/audit/templates/audit/audit-info-form.html

```diff
@@ -69,10 +68,9 @@ <h1>Financial statements</h1>
                             <input id="is_sp_framework_required--true"
                                 name="is_sp_framework_required"
                                 class="usa-radio__input"
-                                required
                                 type="radio"
                                 value="True"
-                                {% if form.cleaned_data.is_sp_framework_required is True %}checked{% endif %} />
```

Comment:
We should be using `is True` here unless we intend to be checking against a non-boolean value that we want to be coerced into True/False.

Reply:
Agreed. I missed this one.